src/main/cpp/subsystems/DriveSubsystem.cpp

```diff
@@ -73,7 +73,7 @@ void DriveSubsystem::SimulationPeriodic() {
   m_gyroSimAngle.Set(m_gyro.GetAngle() +
                      (chassisSpeeds.omega.convert<units::deg_per_s>().value() *
                       DriveConstants::kLoopTime.value()));
-  poseEstimator.Update(-m_gyro.GetRotation2d(), GetModulePositions());
+  poseEstimator.Update(-GetRotation(), GetModulePositions());
```

Comment:
What’s the difference between these calls?

Reply:
removing copy-paste